Lib/ntpath.py

```diff
@@ -540,6 +531,12 @@ def expandvars(path):
 try:
     from nt import _path_normpath
 
+    def normpath(path):
```

Comment:
Why has this function been moved?

Reply:
It's shorter.

Reply:
That is insufficient reason to make a change, please revert.

Reply:
Yes, I understand.